app/models/applicant.rb

```diff
@@ -21,6 +21,10 @@ class Applicant < ApplicationRecord
   scope :for_current_workflow, -> { where(application_token: Rails.application.credentials.alloy[:token]) }
 
   validates :first_name, :last_name, :ssn, :birthdate, presence: true
+  
+  include PgSearch::Model
+  pg_search_scope :global_search,
+    against: [:first_name, :last_name, :ssn, :birthdate]
```

Comment:
what's the attribute for the claim #? I tried :case_number but that wasn't quite right. :S

Reply:
`id`. When I do rails apps, I change the default id column from incremental integer to postgres UUID.